src/kbc_modules/offline_fs_kbc/README.md

```diff
@@ -27,20 +27,30 @@ Resources must be provide in the guest file system at `/etc/aa-offline_fs_kbc-re
 {
   "Policy": "<base64-encoded content from policy.json>",
   "Sigstore Config": "<base64-encoded content from sigstore_config.yaml>",
-  "GPG Keyring": "<base64-encoded content from pubkey.gpg>"
+  "GPG Keyring": "<base64-encoded content from pubkey.gpg>",
+  "Cosign Key": "<base64-encoded content from cosign.gpg>",
```

Comment:
`cosign.gpg` -> `cosign.pub`. In fact, the key used here is an ecdsa-p256 public key with SPKI format rather than a gpg key

Reply:
fixed